modules/tinymce/changelog.txt

```diff
@@ -14,6 +14,7 @@ Version 5.7.0 (TBD)
     Fixed an issue where the image dialog tried to calculate image dimensions for an empty image URL #TINY-6611
     Fixed an issue where `scope` attributes on table cells would not change as expected when merging or unmerging cells #TINY-6486
     Fixed the plugin documentation links in the `help` plugin #DOC-703
+    Fixed events bound using `DOMUtils` not returning the correct result for `isDefaultPrevented` in some cases #TINY-6834
```

Comment:
Should we mention the `Dropped file type not supported` problem as well?

Reply:
It was more of a side effect that was showing, but given it was from a support case I guess it can't hurt :shrug: